### ISSUE TITLE
TimeZoneInfo raises KeyError if Windows registry contains a TZ without a Std key

### ISSUE THREAD

**Issue by buchi** [mhammond/pywin32#1299]
I'm not sure why some Windows installations contain timezone registry entries without a `Std` key.
However this seems to be the case in some rare situations.

Traceback:
```
Traceback (most recent call last):
  File "site-packages\win32\lib\win32timezone.py", line 698, in utc
  File "site-packages\win32\lib\win32timezone.py", line 475, in __init__
  File "site-packages\win32\lib\win32timezone.py", line 496, in _LoadInfoFromKey
  File "site-packages\win32\lib\win32timezone.py", line 482, in _FindTimeZoneKey
  File "site-packages\win32\lib\win32timezone.py", line 723, in get_index_value
KeyError: 'Std'
```

To reproduce the problem, delete the Std key of a timezone registry entry (e.g. HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Time Zones\UTC\Std) and call `win32timezone.TimeZoneInfo.utc()`.

Python version: 3.6.8, pywin32 224

Also seen here: https://sourceforge.net/p/pywin32/mailman/message/28751500/

**Comment by mhammond** [comment]
@jaraco, any thoughts on this?

**Comment by jaraco** [comment]
I've never had a system that exhibited this behavior. I don't understand from where the irregularity originates. Nevertheless, a crash is undesirable. What I'd like to see is what other APIs (namely those supplied by MS) do under these conditions. [These docs](https://docs.microsoft.com/en-us/windows/desktop/api/timezoneapi/ns-timezoneapi-_time_zone_information) suggest that the standard name and daylight name can be empty. Maybe if the field is missing from the registry, the empty string is used as a fallback, implying that
```python
	def _LoadInfoFromKey(self):
		...
		self.standardName = key.get('Std', '')
		self.daylightName = key.get('Dlt', '')
		...
```
might be the proper fix.

@buchi: On your machine, what do you get for

```
win32timezone.TimeZoneDefinition.current().standardName
```

When your current time zone is missing the Std key?

**Comment by buchi** [comment]
@jaraco: The error doesn't happen on my machine, but in the environment of one of our clients.

I've located the following timezone entry causing the error:
```
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Time Zones\Mexico Standard Time]
"Index"=dword:0000001e
```

No idea where this entry comes from (the client is located in Switzerland, Europe), but it's missing most of the values (Std, Dlt, TZI, ...).

I can't run your suggested command for this timezone as I can't switch to it.
I've also tried deleting the Std/Dlt values from my current timezone, but I still get the same result as before.

I would suggest to just ignore incomplete time zone entries.
